File: IonicSolution/Webserver/01052021/ServerDeviceWeb/models/devicedetail.ts
import { Byte } from "@angular/compiler/src/util"

export class DeviceDetail {

    strDeviceId: string;
    strDeviceIds: string;
    strPassword: string;
    strVolume: number;
    strShowDubugPanel: boolean;
    boolUseSampling: boolean;
    boolIsOnline: boolean;
    bytFaceDetectionQuality: number;
    bytFaceDetectionMode: number;
    ushortFaceDetectionFrameRateMaximum: number;
    boolQrCodeActive: boolean;
    byteDetectionCount: number;
    dateClientDateTimeTicks: Date
    dateServerDateTimeTicks: Date
    constructor() {
        this.strDeviceId = '';
        this.strDeviceIds = '';
        this.strPassword = '';
        this.strVolume = 0;
        this.strShowDubugPanel = false;
        this.boolUseSampling = false;
        this.boolIsOnline = false;
        this.bytFaceDetectionQuality = 0;
        this.bytFaceDetectionMode = 0;
        this.ushortFaceDetectionFrameRateMaximum = 0;
        this.boolQrCodeActive = false;
        this.byteDetectionCount = 0;
        this.dateClientDateTimeTicks = new Date();
        this.dateServerDateTimeTicks = new Date();
    }
}

File: IonicSolution/Webserver/01052021/ServerDevice/Models/DeviceDetail.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace ServerDevice.Models
{
    public class DeviceDetail
    {
        [Key]
        public string strDeviceId { get; set; }
        public string strDeviceIds { get; set; }
        public string strPassword { get; set; }
        public float strVolume { get; set; }
        public bool strShowDubugPanel { get; set; }
        public bool boolUseSampling { get; set; }
        public bool boolIsOnline { get; set; }
        public byte bytFaceDetectionQuality { get; set; }
        public byte bytFaceDetectionMode { get; set; }
        public ushort ushortFaceDetectionFrameRateMaximum { get; set; }
        public bool boolQrCodeActive { get; set; }
        public byte byteDetectionCount { get; set; }
        public DateTime dateClientDateTimeTicks { get; set; }
        public DateTime dateServerDateTimeTicks { get; set; }
    }
}
